plugins/autopeering/autopeering.go

```diff
@@ -25,16 +24,16 @@ import (
 
 // autopeering constants
 const (
-	ProtocolVersion = 0      // update on protocol changes
-	NetworkVersion  = "v0.2" // update on network changes
+	ProtocolVersion = 0 // update on protocol changes
+	NetworkVersion  = 2 // update on network changes
 )
 
 var (
 	// ErrParsingMasterNode is returned for an invalid master node.
 	ErrParsingMasterNode = errors.New("cannot parse master node")
 
 	// networkID specifies the autopeering network identifier.
-	networkID = hash32([]byte(banner.AppVersion + NetworkVersion))
+	networkID = hash32([]byte{NetworkVersion})
```

Comment:
That is now a bit strange, we just need a unit32 for the networkID.
If the networkVersion is already a number we don't need to hash it.
Do we want to add something like `-develop` or `-snapshot` in the future?